pkg/schedule/labeler/labeler.go

```diff
@@ -88,11 +89,10 @@ func (l *RegionLabeler) checkAndClearExpiredLabels() {
 			continue
 		}
 		if len(rule.Labels) == 0 {
-			err = l.storage.DeleteRegionRule(key)
-			delete(l.labelRules, key)
+			err = l.DeleteLabelRuleLocked(key)
 			deleted = true
 		} else {
-			err = l.storage.SaveRegionRule(key, rule)
+			err = l.SetLabelRuleLocked(rule)
```

Comment:
I prefer not to change the unnecessary code, it does one more check and memory modification.